sn_logging/src/appender.rs

```diff
@@ -33,7 +33,7 @@ use tracing_appender::non_blocking::{NonBlocking, WorkerGuard};
 /// - older files are deleted.
 pub(super) fn file_rotater(
     dir: &PathBuf,
-    max_lines: usize,
+    max_bytes: usize,
```

Comment:
Just a nitpick, the docs above might need updating too?

Reply:
Ah, good spot! Changed. Thanks.